configs/body_factory/default/request#uri_len_too_long

```diff
@@ -0,0 +1,15 @@
+<HTML>
+<HEAD>
+<TITLE>URI Too Long</TITLE>
+</HEAD>
+
+<BODY BGCOLOR="white" FGCOLOR="black">
+<H1>URI Too Long</H1>
+<HR>
+
+<FONT FACE="Helvetica,Arial"><B>
+Description: Could not process this request because
+the request uri was too long (> 28K bytes).
```

Comment:
Where does the`28K bytes` come from?

Reply:
Oops..it was something I was testing with locally and planning on using for our prod. Will remove that. Good catch!